src/PIL/ImageDraw.py

```diff
@@ -542,7 +543,8 @@ def draw_text(ink, stroke_width=0, stroke_offset=None):
                 # font.getmask2(mode="RGBA") returns color in RGB bands and mask in A
                 # extract mask and set text alpha
                 color, mask = mask, mask.getband(3)
-                color.fillband(3, (ink >> 24) & 0xFF)
+                ink_alpha = struct.pack("=i", ink)[3]
```

Comment:
```suggestion
                ink_alpha = struct.pack("i", ink)[3]
```
Searching through Pillow's existing code, the tradition has been to use native mode implicitly rather than explicitly.
https://github.com/python-pillow/Pillow/blob/76446ee45066554380e08b2f0c4c514f43a297c1/src/PIL/SpiderImagePlugin.py#L259
https://github.com/python-pillow/Pillow/blob/76446ee45066554380e08b2f0c4c514f43a297c1/src/PIL/SgiImagePlugin.py#L184-L188

Reply:
The default is `@` mode, which is not equivalent to `=`.
From https://docs.python.org/3/library/struct.html#byte-order-size-and-alignment:

> Note the difference between '@' and '=': both use native byte order, but the size and alignment of the latter is standardized.

The `s` and `f` formats are the same for both `@` and `=`.
`l` is platform dependent with `@`, but is used with `>` in the quoted code above.

`i` is platform dependent, so changing `=` to `@` can change the behavior here.
However, in the C code, I see that `int` is assumed to be at least 4 bytes, so it is likely the same on all currently supported platforms even if it is not the same in general.

Reply:
I hadn't read the linked documentation in enough detailed to notice the distinction before I made my suggestion. Thanks nulano for catching that.

I don't have strong feelings, I was just trying to follow a convention. If there is a preference to future proof this code by using `=`, that's fine with me.

Reply:
Makes sense, @nulano let us know if you want to be sure to include this

Reply:
Thanks @nulano, just marked this thread as unresolved.